contrib/babelfishpg_common/src/collation.c

```diff
@@ -1167,6 +1185,17 @@ get_collation_oid_internal(char *collation_name)
 Oid
 get_server_collation_oid_internal(bool missingOk)
```

Comment:
same as above, please update the name of this helper to reflect it may return database or server collation